### PR TITLE
Event subscription with aggregates and fabrics sections

### PR DESCRIPTION
**Pull request by sunithasomanna**
Added sections for Event subscription with aggregates and fabrics in the API readme

### CODE REVIEW

**Review by chauberahul1993 — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -10651,7 +10653,7 @@ To know the progress of this action, perform HTTP `GET` on the *[task monitor](#
 To get the list of subtask URIs, perform HTTP `GET` on the task URI returned in the JSON response body. See *Sample response body (HTTP 202 status)*. The JSON response body of each subtask contains a link to the task monitor associated with it. To know the progress of this operation (subtask) on a specific server, perform HTTP `GET` on the task monitor associated with the respective subtask.
 
 
->**curl command**
+>**curl command to create event subscriptions for systems**
```
Comment (amar-shalgar) [inline docs/README.md]: subscriptions can be created for multiple other things. i feel systems should not be here

File: docs/README.md [hunk 2]
```diff
@@ -10689,7 +10691,7 @@ curl -i POST \
 ```
 
 
->**Sample request body**
+>**Sample request body to create event subscriptions for systems**
```
Comment (amar-shalgar) [inline docs/README.md]: subscriptions can be created for multiple other things. i feel systems should not be here

File: docs/README.md [hunk 3]
```diff
@@ -10718,10 +10720,149 @@ curl -i POST \
...
+        "@odata.id":"/redfish/v1/Aggregates/{AggregateId}"
+      },
+      {
+        "@odata.id":"/redfish/v1/Aggregates/{AggregateId}"
+      }
+   ]
+}
+```
+
+>**curl command to create event subscriptions for fabrics**
```
Comment (amar-shalgar) [inline docs/README.md]: Should we add single subscription request and mention what other OriginResources can be provided

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (sunithasomanna) [reply docs/README.md]: Addressed comment

File: docs/README.md (hunk 2 above)
Comment (sunithasomanna) [reply docs/README.md]: Addressed comment

File: docs/README.md (hunk 3 above)
Comment (sunithasomanna) [reply docs/README.md]: added Aggregates to the table in Origin Resources section

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**